frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -0,0 +1,95 @@
+  cy.byLegacyTestID('operator-install-btn').click({ force: true });
+  cy.byTestID('Operator recommended namespace:-radio-input').should('be.checked');
+  cy.byTestID('enable-monitoring').click();
+  cy.byTestID('install-operator').click();
+  cy.byTestID('success-icon', { timeout: 180000 }).should('be.visible');
+  cy.exec('oc get project openshift-storage -o json').then((res) => {
+    const obj = JSON.parse(res.stdout);
+    expect(obj.metadata.labels?.['openshift.io/cluster-monitoring']).toEqual('true');
+  });
+  // Rook, Noobaa and OCS pod should come up after installation.
+  cy.exec('oc get po -n openshift-storage -o json').then((res) => {
+    const { items } = JSON.parse(res.stdout);
+    expect(items.find((item) => _.startsWith(item.metadata.name, 'noobaa-operator'))).toBeDefined();
+    expect(items.find((item) => _.startsWith(item.metadata.name, 'ocs-operator'))).toBeDefined();
+    expect(
+      items.find((item) => _.startsWith(item.metadata.name, 'rook-ceph-operator')),
+    ).toBeDefined();
+  });
+
+  // Make changes to this once we add annotation
+  cy.log(`Install OCS in ${mode} Mode`);
+  cy.clickNavLink(['Installed Operators']);
+  cy.byLegacyTestID('item-filter').type('ocs-operator');
+  cy.byTestOperatorRow('OpenShift Container Storage').click();
+  cy.byLegacyTestID('horizontal-link-Storage Cluster').click();
+  cy.byTestID('yaml-create').click();
+
+  cy.log(`Select ${mode}`);
+  cy.byTestID('Internal-radio-input').should('be.checked');
+
+  // Step 1
+  // Select all worker Nodes
+  cy.get('input[name=check-all]').check();
+  cy.get('input[name=check-all').should('be.checked');
+  // Two dropdowns in the same page.
+  // (Todo: )make dropdown data-test-id be something that can be passed as a prop
+  cy.byLegacyTestID('dropdown-button')
+    .first()
+    .click();
+  cy.byTestDropDownMenu('512Gi').click();
+  cy.get('.pf-c-button.pf-m-primary')
+    .contains('Next')
+    .click();
+
+  // Step 2
+  if (encrypted) {
+    cy.log('Enabling Encryption');
+    cy.byTestID('encryption-checkbox').click();
+  }
+  cy.get('.pf-c-button.pf-m-primary')
+    .contains('Next')
+    .click();
+
+  // Final Step
+  cy.get('.pf-c-button.pf-m-primary')
+    .contains('Create')
+    .click();
+
+  // Check worker nodes for labels
+  cy.exec('oc get nodes -o json').then((res) => {
+    const { items } = JSON.parse(res.stdout);
+    items
+      .map((item) => item.metadata.labels)
+      .filter((item) => item.hasOwnProperty('node-role.kubernetes.io/worker'))
+      .forEach((item) =>
+        expect(item.hasOwnProperty('cluster.ocs.openshift.io/openshift-storage')).toBeTruthy(),
+      );
+  });
+
+  // Wait for the storage cluster to reach Ready
+  // Storage Cluster CR flickers so wait for 10 seconds
+  // Disablng until ocs-operator fixes above issue
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(10000);
+
+  return cy.byTestID('resource-status').contains('Ready', { timeout: 900000 });
```

Comment:
We're expecting to wait 15 minutes for a Ready status?  Is there a cy.exec('oc ...') cmd we can call to check status?   Something like [this](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/support/resources.ts#L32)?

Reply:
Again similar to checking if the Operator is installed, I think this would be better than to poll the CLI.

Reply:
Ok, I think we might have to come up with a better solution, we'll see how often this test 'flakes'.